CRM/Upgrade/Incremental/sql/5.13.alpha1.mysql.tpl

```diff
@@ -1 +1,5 @@
 {* file to handle db changes in 5.13.alpha1 during upgrade *}
+
+-- dev/core#829 Swaziland has changed its name to Eswatini
+UPDATE civicrm_country SET name = 'Eswatini' WHERE ID = 1203';
```

Comment:
You need to remove the single quote after 1203

Reply:
@lordt we've gotten in trouble in the past where the ids are non-standard - instead of where id = 1203 can you do where name = 'Swaziland'?